tests/test_logsub.py

```diff
@@ -23,22 +19,9 @@
 class TestLogsub(TestCase):
 
     def setUp(self):
-        self.old_dir = os.getcwd()
-        os.chdir(tempfile.gettempdir())
```

Comment:
this tempfile stuff is fishing, it's failing 1 out of a couple dozen test runs. the only thing we're testing is a file is being written to disk, the more important things seem to be the methods as part of the log_sub package, like message_event